src/platform/plugins/shared/discover/public/context_awareness/profile_providers/security/accessors/get_row_additional_control.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import type { RowControlColumn } from '@kbn/discover-utils';
+import type { ProfileProviderServices } from '../../profile_provider_services';
+import { ExploreInSecurity } from '../components/row_leading_controls/explore_in_security';
+
+export function getRowAdditionalLeadingControls({
+  services,
+  additionalControls,
+}: {
+  services: ProfileProviderServices;
+  additionalControls: RowControlColumn[];
+}) {
+  const createExploreInSecurityControl = (): RowControlColumn => {
+    return {
+      id: 'exploreInSecurity',
+      headerAriaLabel: 'Explore in Security',
```

Comment:
Do we want to translate this text?

Reply:
Added needed translations texts. Thank you. Requesting you to review again.